public/asset/js/ContactModalForm.js

```diff
@@ -0,0 +1,86 @@
+document.addEventListener("DOMContentLoaded", function () {
+    let formContactModal = document.getElementById('formContactModal');
+    formContactModal.addEventListener('show.bs.modal', showFormContactModal);
+
+    let btnSaveForm = document.getElementById('btnSaveForm');
+    btnSaveForm.addEventListener('click', saveForm);
+
+    var formContact = document.getElementById('formContact');
+});
+
+function saveForm() {
+    let formData = new FormData(formContact);
+    let plainFormData = Object.fromEntries(formData.entries());
+    let formDataJsonString = JSON.stringify(plainFormData);
+
+    const fetchOptions = {
+        method: "POST",
+        headers: {
+            "Content-Type": "application/json",
+            "Accept": "application/json",
+            "Authorization": "Bearer ".concat(api_token)
+        },
+        body: formDataJsonString,
+    };
+
+    fetch(formContact.action, fetchOptions)
+        .then((response) => response.json())
+        .then((data) => {
+
+            console.log(data);
+
+            if (data.errors) {
+                // TODO
+                alert(Object.values(data.errors));
+            }
+
+            // TODO
+            alert('The contact has been saved successfully.');
```

Comment:
Please remove the alerts, change for console.log or add another way to show a message like we do with Twitter Boostrap Toast for notifications